tests/mcuboot/Makefile

```diff
@@ -0,0 +1,15 @@
+APPLICATION = hello_mcuboot
+
+BOARD ?= nrf52dk
+
+include ../Makefile.tests_common
+
+BOARD_WHITELIST := nrf52dk
+
+export IMAGE_VERSION = 1.1.1+1
```

Comment:
This variable is needed for imgtool, which only allows this kind of versioning format.